inc/saitam.h

```diff
@@ -1289,6 +1289,12 @@ typedef enum _sai_tam_report_mode_t
     /** Report in a bulk mode */
     SAI_TAM_REPORT_MODE_BULK,
 
+    /** Report in a probabilistic sampling mode, one report is sent in n events */
+    SAI_TAM_REPORT_MODE_PROBABILISTIC_SAMPLING,
+
+    /** Report in a microburst mode, specifying report rate and max number of reports per microburst event */
```

Comment:
If the objective here is to rate-limit the number of reports, then attaching a policer object to tam_report will be more intuitive.

Reply:
Base on last community discussion, all policer attributes may not be applicable and we don't have capability query mechanism based on policer bind point to identify which of these attributes are supported.
We will follow existing SAI attribute names instead of using microburst.

Reply:
Please update the document also to remove references to microburst and also use the updated attribute names.

Reply:
Thanks @ashutosh-agrawal . I missed to replace the HLD file. Now I have pushed the changes. Pls review.